### PR TITLE
Check for null workload packs

### PR DESCRIPTION
**Pull request by sfoslund**

### CODE REVIEW

**Comment by radical** [comment]
Should a test be added for this?

File: src/Assets/TestProjects/SampleManifest/Sample.json
Comment (sfoslund) [inline src/Assets/TestProjects/SampleManifest/Sample.json:159]: Adding a fake pack with no appropriate alias to the manifest used for testing, this causes the issue to repro in several existing tests.

Comment (wli3) [reply src/Assets/TestProjects/SampleManifest/Sample.json:159]: Is that an authoring error? why it points to a non existing pack?

Comment (sfoslund) [reply src/Assets/TestProjects/SampleManifest/Sample.json:159]: This is simulating a pack that doesn't have a alias for the current rid, except for testing purposes it's helpful for this pack to not have a valid alias for any rid's.

**Review by lewing — APPROVED** [review]

**Comment by sfoslund** [comment]
@wli3 @joeloff can I get a quick review from someone on our team here before I merge?

**Review by dsplaisted — APPROVED** [review]

**Comment by radical** [comment]
This will be merged into `main` too, correct?

**Comment by sfoslund** [comment]
> This will be merged into main too, correct?

Yes, I plan on merging everything from preview 5 into main at once some time this week.

**Comment by lewing** [comment]
Thanks @sfoslund some reporting infrastructure is ready to take advantage of it.

**Comment by sfoslund** [comment]
It was merged as part of https://github.com/dotnet/sdk/pull/18026 earlier today so builds should be available soon.

**Pull request merged**
